depends/packages/zeromq.mk

```diff
@@ -1,9 +1,9 @@
 package=zeromq
-$(package)_version=4.3.4
+$(package)_version=4.3.5
```

Comment:
nit-28627: doc/dependencies.md is not updated but there's no backport with these changes, I guess someday